### PR TITLE
Bugfix/double label radio

### PR DESCRIPTION
**Pull request by 01231**
Fixes #(issue).

# Done is when (DoD):
- [ ] I have added UI/Unit tests for my changes
- [ ] I added modified component properties to the typescript typings
- [ ] Design has been reviewed with (here name of reviewer)
- [ ] My changes are accordingly to our [Best Practices](https://github.com/axa-ch/patterns-library/blob/develop/CONTRIBUTION.md#best-practices)
- [ ] Old tests and eslint rule are not broken
- [ ] My changes are well documented in the README and showcased in the stories 
- [ ] Tested with IE

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas (focus on **WHY**)
- [ ] I have made/updated the ChangeLog Section in the README

### CODE REVIEW

File: src/components/10-atoms/radio/CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,17 @@
 # Changelog
 
+### 2.0.2
+
+prevent doubled labels when `label` attribute isn't set
```
Comment (raphaellueckl) [inline src/components/10-atoms/radio/CHANGELOG.md]: `Prevent doubled labels when `label` attribute is not set.`

File: src/components/10-atoms/radio/demo.js [hunk 2]
```diff
@@ -81,36 +81,45 @@ storiesOf('Atoms/Radio/Demos', module)
...
     return root;
-  });
+  })
+  .add(
+    'Radio - no label',
+    () => `
+      <axa-fieldset>
+        <p>This axa-radio's have no labels. Therefore no label will be rendered</p>
+        <axa-radio id="radio1" name="insurance"></axa-radio>
+        <axa-radio id="radio2" name="insurance" label=""></axa-radio>
```
Comment (raphaellueckl) [inline src/components/10-atoms/radio/demo.js]: Does this need an extra story?

File: src/components/10-atoms/radio/index.js [hunk 3]
```diff
@@ -159,29 +159,38 @@ class AXARadio extends NoShadowDOM {
...
     // controlledness is a React-only notion
     this.state.isControlled = isControlled && isReact;
+    const inputElement = html`
+      <input
+        id="${refId}"
+        class="a-radio__input"
+        type="radio"
+        name="${name}"
+        ?checked="${checked}"
+        value="${value}"
```
Comment (raphaellueckl) [inline src/components/10-atoms/radio/index.js:168]: For better grouping, I would move `value` 1 line up.

File: src/components/10-atoms/radio/demo.js (hunk 2 above)
Comment (01231) [reply src/components/10-atoms/radio/demo.js]: I need it to test it...

File: src/components/10-atoms/radio/index.js (hunk 3 above)
Comment (01231) [reply src/components/10-atoms/radio/index.js:168]: ✅

File: src/components/10-atoms/radio/CHANGELOG.md (hunk 1 above)
Comment (01231) [reply src/components/10-atoms/radio/CHANGELOG.md]: ✅

File: src/components/10-atoms/radio/demo.js (hunk 2 above)
Comment (danyball) [reply src/components/10-atoms/radio/demo.js]: You can test individually with knobs. Please have a look if you can build a knob for that option.

File: src/components/10-atoms/radio/ui.test.js [hunk 4]
```diff
@@ -81,3 +81,22 @@ test('should show controlled-component behaviour in default mode', async t => {
 
   await t.expect(radio1.getAttribute('checked')).eql('');
 });
+
+fixture('Radio - no label').page(
+  `${host}/iframe.html?id=atoms-radio-demos--radio-no-label`
```
Comment (danyball) [inline src/components/10-atoms/radio/ui.test.js]: The test-url with knobs can be like this example:

  `${host}/iframe.html?id=molecules-datepicker--datepicker&knob-inputfield=true&knob-locale=de-CH&knob-year=2020&knob-month=4&knob-day=22&knob-label=&knob-labelbuttoncancel=Cancel&knob-labelbuttonok=Ok&knob-monthtitle=Choose%20Month&knob-yeartitle=Choose%20Year&knob-invaliddatetext=Invalid%20date&knob-placeholder=Please%20select%20a%20date&knob-width=227&knob-height=80&knob-width%20(note%20max-length)=200`

File: src/components/10-atoms/radio/CHANGELOG.md [hunk 5]
```diff
@@ -1,13 +1,17 @@
 # Changelog
```
Comment (danyball) [inline src/components/10-atoms/radio/CHANGELOG.md]: We dont need that headline

**Review by danyball — CHANGES_REQUESTED** [review]

File: src/components/10-atoms/radio/demo.js [hunk 6]
```diff
@@ -81,36 +81,45 @@ storiesOf('Atoms/Radio/Demos', module)
...
+    `;
 
     render(template, root);
     return root;
-  });
+  })
+  .add(
+    'Radio - no label',
+    () => `
+      <axa-fieldset>
```
Comment (raphaellueckl) [inline src/components/10-atoms/radio/demo.js]: https://axa-ch.github.io/plib-feature/bugfix/double-label-radio/?path=/story/atoms-radio-demos--radio-no-label

This example does not work.

File: src/components/10-atoms/radio/CHANGELOG.md (hunk 5 above)
Comment (raphaellueckl) [reply src/components/10-atoms/radio/CHANGELOG.md]: Why? I would keep it.

File: src/components/10-atoms/radio/demo.js (hunk 2 above)
Comment (01231) [reply src/components/10-atoms/radio/demo.js]: The problem is, that I can't pass an empty label with the knobs. This results in the default label. In our case `Ja`.

File: src/components/10-atoms/radio/demo.js (hunk 6 above)
Comment (01231) [reply src/components/10-atoms/radio/demo.js]: True, I will check it out.

File: src/components/10-atoms/radio/demo.js (hunk 2 above)
Comment (raphaellueckl) [reply src/components/10-atoms/radio/demo.js]: Are you sure that this is not possible? `&knobname=&somethingelse=hi`

Comment (01231) [reply src/components/10-atoms/radio/demo.js]: only works when there is no default value set, but I need a default value :(

File: src/components/10-atoms/radio/ui.test.js (hunk 4 above)
Comment (01231) [reply src/components/10-atoms/radio/ui.test.js]: As discussed with raphi in the daily, it is not possible in this case. Therefore I will close this conversation.

File: src/components/10-atoms/radio/demo.js (hunk 2 above)
Comment (01231) [reply src/components/10-atoms/radio/demo.js]: As discussed with raphi in the daily, it is not possible in this case. Therefore I will close this conversation.

File: src/components/10-atoms/radio/demo.js (hunk 6 above)
Comment (01231) [reply src/components/10-atoms/radio/demo.js]: I deleted the demo.

File: src/components/10-atoms/radio/story.js [hunk 7]
```diff
@@ -39,12 +39,43 @@ storiesOf('Atoms/Radio', module)
...
+          value="2"
+        ></axa-radio>
+        <axa-radio
+          name="contract"
+          icon="${icon ? iconSVG : ''}"
+          ?noGap="${noGap}"
+          ?button="${button}"
+          ?disabled="${disabled}"
+          ?noAutoWidth="${noAutoWidth}"
+          label="Vielleicht"
```
Comment (raphaellueckl) [inline src/components/10-atoms/radio/story.js]: german

File: src/components/10-atoms/radio/story.js [hunk 8]
```diff
@@ -39,12 +39,43 @@ storiesOf('Atoms/Radio', module)
...
+          value="1"
+        ></axa-radio>
+        <axa-radio
+          name="contract"
+          icon="${icon ? iconSVG : ''}"
+          ?noGap="${noGap}"
+          ?button="${button}"
+          ?disabled="${disabled}"
+          ?noAutoWidth="${noAutoWidth}"
+          label="Nein"
```
Comment (raphaellueckl) [inline src/components/10-atoms/radio/story.js]: german

Comment (01231) [reply src/components/10-atoms/radio/story.js]: done

File: src/components/10-atoms/radio/story.js (hunk 7 above)
Comment (01231) [reply src/components/10-atoms/radio/story.js]: done

**Review by raphaellueckl — APPROVED** [review]

**Pull request merged**